about.html

```diff
@@ -0,0 +1,118 @@
+      </div>
+    </section>
+    <section class="logoInfo" id="sectionLink1">
+      <div class="logoDescription">
+        <h2 class="logoDescriptionTitle">Cordoba<br class="breakMobile"/> Clasical Spring Festival Logo</h2>
+        <p class="logoDescriptionText">The logo of Cordoba Clasical Spring Festival was decided <br/>
+          through the logo competition from 8, june to 7, july.
+        </p>
+        <p class="logoDescriptionTextDesktop">The logo of Cordoba Clasical Spring Festivalwas decided through the logo
+          competition from 8.June to 7.July.<br/>After online public voting and Selection Comittee's judging 
+          process, the logo from Naresh Agrwal won the prize. <br/>After consultation with Mr.Agrawal we optimized 
+          the prize-wining-logo for this year's summit as following:
+        </p>
+      </div>
+      <div class="logoDiv">
+        <img src="./assets/navbar/orchestraIcon.jpg" class="logoInfoLogo" alt="Cordoba Spring Festival 2023 Icon">
+      </div>
+    </section>
+    <section class="pastConcerts" id="sectionLink2">
+      <h2 class="pastConcertTitle">See the past Cordoba Clasical Spring Festivals</h2>
+      <p class="pastConcertDescription">Take a look at the last two Cordoba Spring Festivals which took place <br/>
+      in Cordoba</p>
+      <div class="pastConcertsDiv">
+        <div class="concertContainer">
+          <div class="redBackground"></div>
+          <img  class="pastConcertsImg" src="./assets/pastConcerts/pastConcertC.jpg" alt="Cordoba Spring Festival 2022 Icon">
+          <div class="concertText">2021 <br/><span class="concertTextResize">Cordoba Clasical Spring Festival</span></div> 
+        </div>
+        <div class="concertContainer">
+          <div class="redBackground"></div>
+          <img  class="pastConcertsImg" src="./assets/pastConcerts/pastConcertC.jpg" alt="Cordoba Spring Festival 2022 Icon">
+          <div class="concertText">2022 <br/><span class="concertTextResize">Cordoba Clasical Spring Festival</span></div> 
+        </div>
+      </div>
+    </section>
+  </main>
+  <footer class="aboutFooter">
+    <article class="sponsors aboutDisplay">
+      <div class="sponsorTitleDiv">
+        <p class="sponsorTitle">Partner</p>
+      </div>
+      <ul class="sponsorList">
+        <li class="sponsorListItem"><img src="./assets/footer/mozillaIcon.png" class="sponsorIcons " alt="Mozilla Icon"></li>
+        <li class="sponsorListItem"><img src="./assets/footer/googleIcon.png" class="sponsorIcons iconMargin" alt="Google Icon"></li>
+        <li class="sponsorListItem"><img src="./assets/footer/naverIcon.png" class="sponsorIcons" alt="naver Icon"></li>
+        <li class="sponsorListItem"><img src="./assets/footer/daumkakaoIcon.png" class="sponsorIcons iconMargin" alt="daumkakao Icon"></li>
+        <li class="sponsorListItem"><img src="./assets/footer/airbnbIcon.png" class="sponsorIcons iconMargin iconSize" alt="airbnb Icon"></li>
+      </ul>
+    </article>
```

Comment:
- [ ] Same here as pointed out earlier :point_up: kindly remove the partner section from the `footer` tag